### PR TITLE
feat: add redis config

### PR DESCRIPTION
**Pull request by oriohol95**

### CODE REVIEW

File: packages/signaling-server/src/index.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
 const socketIO = require('socket.io')
+const redis = require('socket.io-redis')
 
 const createRoomPool = require('./createRoomPool')
 const Node = require('./Node')
 
-function SignalingServer () {
+function SignalingServer ({ redisHost, redisPort }) {
```
Comment (d-asensio) [inline packages/signaling-server/src/index.js:7]: I would group the properties, something like

```js
const config = {
   redis: {
     host: '...',
     port: 4747
   }
}

const server = SignalingServer(config)
```

**Review by d-asensio — COMMENTED** [review]
I suggest to this using a config file. You could use [cosmoconfig](https://github.com/davidtheclark/cosmiconfig#readme) or [rc](https://www.npmjs.com/package/rc) for example.

This way the user could choose how to load the configuration.

Great job BTW!

File: packages/signaling-server/src/index.js (hunk 1 above)
Comment (oriohol95) [reply packages/signaling-server/src/index.js]: Much better 👍

**Comment by oriohol95** [comment]
Done!

**Pull request merged**